indexer/altitude_loader.cpp

```diff
@@ -54,7 +54,7 @@ AltitudeLoader::AltitudeLoader(MwmValue const & mwmValue)
   }
 }
 
-bool AltitudeLoader::HasAltitudes() const { return m_header.m_minAltitude != kInvalidAltitude; }
+bool AltitudeLoader::HasAltitudes() const { return m_reader.get() != nullptr; }
```

Comment:
Sorry, the right repro case is to build an mwm with features, but each feature does not have an altitude.